src/legacy/core_plugins/timelion/public/directives/chart/chart.js

```diff
@@ -17,55 +17,52 @@
  * under the License.
  */
 import panelRegistryProvider from '../../lib/panel_registry';
```

Comment:
please refactor panelRegistry to not use uiRegistry.

- inside `plugin.ts` `TimelionPlugin` define an object `private panelRegistry: Record<string, Panel> = {};`
- in the setup contract expose `registerPanel: (id: string, panel: Panel) => { this.panelRegistry[id] = panel; }`
- in the start contract expose `get: (id: string) => { return this.panelRegistry[id]; }`
- also expose `getAll() => { return { ...this.panelRegistry }; }`

Reply:
https://github.com/elastic/kibana/issues/46770